packages/node/src/utils/near.ts

```diff
@@ -80,31 +82,50 @@ export async function wrapBlock(
     const chunkResult = await api.chunk(chunk.chunk_hash);
     const transactionPromises = chunkResult.transactions.map(
       async (transaction) => {
-        const wrappedTx = await wrapTransaction(api, blockResult, transaction);
+        const exectuionOutcome = await api.txStatusReceipts(
```

Comment:
Im not sure this is right. `txStatusReceipts` gets the receipts for a transactions but they may be over many blocks. So we can end up with data from other blocks.

Theres a couple of problems with this:
* We have data from future blocks
* This is non-deterministic when the project is up to date with the chain then the result of this query could change.


It seems that [chunk details](https://docs.near.org/api/rpc/block-chunk#chunk-details) contains the relevant receipts that we want for the block